vnext/include/ReactUWP/Views/KeyboardEventHandler.h

```diff
@@ -0,0 +1,153 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+#include <optional>
+#include <set>
+#include <folly/dynamic.h>
+#include <winrt/Windows.UI.Xaml.Controls.h>
+#include <winrt/Windows.UI.Xaml.Input.h>
+
+#include "XamlView.h"
+#include <IReactInstance.h>
+
+namespace winrt {
+  using namespace Windows::UI;
+  using namespace Windows::UI::Xaml;
+  using namespace Windows::UI::Xaml::Controls;
+  using namespace Windows::UI::Xaml::Input;
+  using namespace Windows::Foundation;
+  using namespace Windows::UI::Xaml::Media;
+  using namespace Windows::System;
+  using namespace Windows::UI::Core;
+}
+
+using namespace std;
+
+namespace react { namespace uwp {
+  enum class HandledEventPhase
+  {
+    CAPTURING = 1, // match the value with EventPhase in React. EventPhase includes None, Capturing, AtTarget, Bubbling
```

Comment:
This is a modern enum class. Why are the enum values all caps? Quick review of the codebase shows these are generally standard PascalCase.

Reply:
Everything is 'CAPTURING' in facebook's folly project. but I change to PascalCase here just as you suggested.

---
In reply to: [295558736](https://github.com/microsoft/react-native-windows/pull/2623#discussion_r295558736) [](ancestors = 295558736)